Marqeta.Core.Sdk/Generated/Banktransfers/Ach/Transitions/TransitionsRequestBuilder.cs

```diff
@@ -147,7 +147,7 @@ public class TransitionsRequestBuilderGetQueryParameters
             /// <summary>Number of bank transfer transitions to retrieve.</summary>
             [QueryParameter("count")]
             public int? Count { get; set; }
-            /// <summary>Field on which to sort. Use any field in the resource model, or one of the system fields lastModifiedTime or createdTime.Prefix the field name with a hyphen (-) to sort in descending order.Omit the hyphen to sort in ascending order.</summary>
```

Comment:
Gonna be grammar police on this bit! :P

Reply:
This is automatically generated from Marqeta's OpenAPI spec, so there is not much we can do to update the doc. 😅

Reply:
Nooo! They just want to watch the world burn.. xD

Reply:
Ok fair enough!